resources/pdns_authoritative_config.rb

```diff
@@ -34,7 +35,7 @@
 
 property :instance_name, String, name_property: true
 property :launch, Array, default: ['bind']
-property :config_dir, String, default: lazy { default_authoritative_config_directory }
+property :config_dir, String, default: lazy { default_authoritative_config_directory(node['platform_family']) }
```

Comment:
I don't see how this dirtier than having the `node['platform_family']`inside the library?